### PR TITLE
Update transfer-data-manually.md

### PR DESCRIPTION
**Pull request by msbemba**
added links on moving One note files

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2644

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/get-help-with-domains/transfer-data-manually.md [hunk 1]
```diff
@@ -101,3 +101,7 @@ Ask users to [move their email, contacts, tasks, and calendar information](https
 ### OneDrive For Business data:
 
 Ask users to Copy/Sync [OneDrive for Business content to their computer](https://support.microsoft.com/office/59b1de2b-519e-4d3a-8f45-51647cf291cd), and then add it back to their new subscription.
+
+### OneNote 
+
+Ask users to [Backup Onenote](https://support.microsoft.com/office/back-up-notes-f58b34b0-611d-435e-87fa-7942a1767af4?ui=en-us&rs=en-us&ad=us) and [Restore notes from a backup](https://support.microsoft.com/en-us/office/restore-notes-from-a-backup-5daf9cb0-6769-4998-a5de-f044fdd0d831?ui=en-us&rs=en-us&ad=us) to their new subscriptions.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/transfer-data-manually.md]:
```suggestion
Ask users to [Back up OneNote](https://support.microsoft.com/office/back-up-notes-f58b34b0-611d-435e-87fa-7942a1767af4?ui=en-us&rs=en-us&ad=us) and to [Restore notes from a backup](https://support.microsoft.com/en-us/office/restore-notes-from-a-backup-5daf9cb0-6769-4998-a5de-f044fdd0d831?ui=en-us&rs=en-us&ad=us) to their new subscriptions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks.

**Pull request merged**